### PR TITLE
Small bugs

### PR DESCRIPTION
**Pull request by epou**

### CODE REVIEW

File: lib/pages/my_reports_pages/components/reports_list_widget.dart [hunk 1]
```diff
@@ -14,6 +14,10 @@ class ReportsList extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+
+    // Sort the reports by creation_time in descending order
+    reports.sort((a, b) => b.creation_time.compareTo(a.creation_time));
```
Comment (DigitalSeneca) [inline lib/pages/my_reports_pages/components/reports_list_widget.dart:19]: creation_time can be null so I can't compile

Comment (epou) [reply lib/pages/my_reports_pages/components/reports_list_widget.dart:19]: Can I make it non-nullable? The DB is not allowed null for this field, so theoretically it should never be null

Comment (DigitalSeneca) [reply lib/pages/my_reports_pages/components/reports_list_widget.dart:19]: yes, we can try it

Comment (epou) [reply lib/pages/my_reports_pages/components/reports_list_widget.dart]: Fixed in [d900c1d](https://github.com/Mosquito-Alert/Mosquito-Alert-Mobile-App/pull/87/commits/d900c1da902cdc267e354cfbde98ea8d855c065e)

**Review by DigitalSeneca — APPROVED** [review]

**Pull request merged**